metadata/service/aggregator/single.yml

```diff
@@ -6,3 +6,4 @@ parameters:
   heka:
     aggregator:
       enabled: true
+      timezone: ${_param:linux_timezone}
```

Comment:
We can always add a new config parameter later if necessary. In general I think we should minimize the introduction of new config parameters.